Vates/VatesAPI/src/vtkMDHexFactory.cpp

```diff
@@ -120,8 +117,7 @@ void vtkMDHexFactory::doCreate(
       API::IMDNode *box = boxes[i];
       Mantid::signal_t signal_normalized = (box->*normFunction)();
 
-      if (std::isfinite(signal_normalized) &&
-          m_thresholdRange->inRange(signal_normalized)) {
```

Comment:
There is a small improvement (4.26s vs 4.82s) generating the cell blanking array in a vtkStructuredGrid. It's a step in the right direction, but it's a fraction of the time spent generating and rendering the `vtkStructuredGrid` and overall not something one would notice without benchmarking.

I tried doing the same with a MDEventWorkspace, but didn't notice any significant improvement in the splatterplot view.